src/components/CardEvento/index.js

```diff
@@ -1,23 +1,37 @@
 import React  from 'react';
-import { Card, Button, CardImg, CardTitle, CardText, CardDeck, CardSubtitle, CardBody } from 'reactstrap';
+import { Link } from 'react-router-dom';
+
+import { 
+  Card, 
+  Button, 
+  CardImg, 
+  CardTitle, 
+  CardText, 
+  CardDeck, 
+  CardSubtitle, 
+  CardBody 
+} from 'reactstrap';
+
+import users from '../../data/users'
 import './style.css'
-import users from '../../data/users' // arquivo json
 
-const CardEvento = (props) => {
+const CardEvento = () => {
   const eventoList = users[0].eventos;
+
   return (
     <CardDeck className="card-deck">
       {
         eventoList && eventoList.map(e => (
-          // return (
             <Card key={e.id}>
```

Comment:
A lista de Card deve ficar fora do componente, o componente é apenas renderização, exemplo:
No seu arquivo Home vc precisa listar os cards de eventos, logo a minha lista fica em Home, e importo o componente Card que será apenas renderização de props. 

Meu arquivo Home ficaria:
```
eventoList && eventoList.map((evento, index) => <CardEvent key={index} evento={evento} />
````

e esse componente CardEvento só renderiza o objeto da lista que foi passado pra ele por exemplo:
```
<Card>
  <CardImg top width="100%" src={evento.img} alt="Banner com a imagem do workshop/bootcamp" />
  <CardBody>	       
    <CardTitle className="title">{evento.nomeEvento}</CardTitle>	              
    <CardSubtitle className="sub-titulo">WoMakersCode</CardSubtitle>	         
    <CardText>{evento.categoria}</CardText>	            
    <CardText>{evento.descricao}</CardText>	            
      <Link to={`/evento/${e.id}`}>
      <Button>Saber mais</Button>
    </Link>
</Card>
```

Os dados devem vir de cima e os componentes como o CardEvento apenas renderizam o que é passado, a gente chegou a ver isso no pensando em react na 2 aula

https://pt-br.reactjs.org/docs/thinking-in-react.html